src/core/modules/entry.js

```diff
@@ -284,6 +284,14 @@ export default class Entry {
         return this;
     }
 
+    /**
+     * @method includeOwner
+     * @memberOf Entry
+     * @description Include the owner details along with the entry/entries details.
+     * @example stack.ContentType(contentType_uid).Entry(entry_uid).includeOwner().fetch()
+     * @returns {Entry}
+     * @instance
+     */
     includeOwner() {
```

Comment:
Check changelog it might be deprecated in past. @nadeem-cs

Reply:
Thanks for the correction. Marked as deprecated.